src/wlapplication_options.cc

```diff
@@ -134,17 +148,30 @@ struct KeyboardShortcutInfo {
 	const SDL_Keysym default_shortcut;
 	SDL_Keysym current_shortcut;
 	const std::string internal_name;
-	const std::function<std::string()> descname;
+	const std::string descname;
 	std::map<std::string /* tribe */, std::string /* building */> fastplace;
 
 	KeyboardShortcutInfo(const std::set<Scope>& s,
 	                     const SDL_Keysym& sym,
 	                     const std::string& n,
-	                     const std::function<std::string()>& f)
+	                     const std::string& f)
 	   : scopes(s), default_shortcut(sym), current_shortcut(sym), internal_name(n), descname(f) {
 	}
 };
 
+// Aliases are used for help generation to simplify inserting common shortcuts
+struct KeyboardShortcutAlias {
+	const KeyboardShortcut real_shortcut;
+	const std::string descname_override;  // Use gettext_noop() for the init value!
+
+	KeyboardShortcutAlias(const KeyboardShortcut real, const std::string& desc_override = "")
```

Comment:
```suggestion
	KeyboardShortcutAlias(const KeyboardShortcut real, const std::string& desc_override = std::string())
```
just for style consistency :)